HQSmokeTests/testCases/test_01_others.py

```diff
@@ -7,6 +7,15 @@
 """"Contains all test cases that aren't specifically related any menu modules"""
 
 @pytest.mark.smoke
+@pytest.mark.login
+@pytest.mark.dashboard
+@pytest.mark.reports
+@pytest.mark.data
+@pytest.mark.users
+@pytest.mark.applications
+@pytest.mark.webApps
+@pytest.mark.messaging
+@pytest.mark.admin
```

Comment:
I don't think our user has admin rights or can see the menu.

Reply:
Yes, I mentioned it as it was present in the testplan. I will remove it.